### PR TITLE
Gray Field Slug

### PR DESCRIPTION
**Pull request by akbajwa**
hey @vikhyatsingh123 kindly merge my PR please

### CODE REVIEW

**Comment by vikhyatsingh123** [comment]
resolve conflicts first

**Comment by akbajwa** [comment]
@vikhyatsingh123 i resolved all conflicts .please merge my pull request.its the last day for hactoberfest

**Comment by vikhyatsingh123** [comment]
What did you add the code. You should read readme section first.

**Comment by akbajwa** [comment]
@vikhyatsingh123 i added the grey slug card from line 97 .I added the slug according to the readme only

File: index.html [hunk 1]
```diff
@@ -37,6 +37,123 @@
...
+				<div class="col-lg-4 mb-4">
+					<div class="card">
+						<img class="card-img-top" src="images/Charger.png" alt="Bulbasaur" />
+						<div class="card-body">
+							<h5 class="card-title">Charger</h5>
+							<p class="card-text">
+								Charger first debut in The Emperor Strikes Back to charge up
+								Burpy with Levelest to try and take down the Iron Warriror.
+							</p>
+							<a href="https://github.com/vikhyatsingh123" class="btn btn-outline-danger btn-sm">Contributed by - Vikhyat</a>
```
Comment (vikhyatsingh123) [inline index.html:152]: Why there is my name.

File: index.html [hunk 2]
```diff
@@ -37,6 +37,123 @@
...
+                            <h5 class="card-title">Tazerling</h5>
+                            <p class="card-text">Tazerlings are small blue Slugs, with yellow highlights on them. Tazerlings have a strangely shaped pattern on their stomachs which are a lighter blue. On top of that, they have seed-like shapes of yellow. Like all Slugs, Tazerlings have short, stubby arms. They have big brown eyes, with yellow lightning bolts above them in an eyebrow scheme. They have what seems to be three teeth, two big teeth on the sides on the top, and a big tooth under and can fit in between the other two teeth.</p>
+                            <a href="https://github.com/akbajwa" class="btn btn-outline-danger btn-sm">Contributed by - Amanpreet Kaur </a>
+                        </div>
+                    </div>
+                </div>
+	        
+				<div class="col-lg-4 mb-4">
+					<div class="card">
+						<img class="card-img-top" src="images/Charger.png" alt="Bulbasaur" />
```
Comment (vikhyatsingh123) [inline index.html:145]: Alt name and title name is not matching

File: index.html [hunk 3]
```diff
@@ -37,6 +37,123 @@
...
+						<img class="card-img-top" src="images/goon.png" alt="Bulbasaur" />
+						<div class="card-body">
+							<h5 class="card-title">Goon</h5>
+							<p class="card-text">
+								The Goon is an especially evil Goon Doc ghoul, seething with
+								dark energy. When it attacks, the Goon has an eerie way of
+								getting into a target Slugslinger’s head. The Goon is
+								determined to conquer every cavern in Slugterra!
+							</p>
+							<a href="https://github.com/vikhyatsingh123" class="btn btn-outline-danger btn-sm">Contributed by - Vikhyat</a>
```
Comment (vikhyatsingh123) [inline index.html:118]: Here too, my name is written

File: index.html [hunk 4]
```diff
@@ -37,6 +37,123 @@
         <span class="navbar-toggler-icon"></span>
       </button>
 
+
+	<div id="gallery-title"></div>
```
Comment (vikhyatsingh123) [inline index.html:41]: Why did you add this line of code

File: index.html [hunk 5]
```diff
@@ -37,6 +37,123 @@
         <span class="navbar-toggler-icon"></span>
       </button>
 
+
```
Comment (vikhyatsingh123) [inline index.html:40]: You should start writing after 114 line of code

**Review by vikhyatsingh123 — CHANGES_REQUESTED** [review]

File: index.html (hunk 5 above)
Comment (akbajwa) [reply index.html:40]: @vikhyatsingh123 sorry for the inconveniences.I would make PR again
